vxquery-benchmark/src/main/resources/noaa-ghcn-daily/conf/weather_conf.xml

```diff
@@ -0,0 +1,45 @@
+<!--
```

Comment:
This seems to be a specific configuration for your testings. Only include generic examples in the repository.